### PR TITLE
update environment

### PR DESCRIPTION
**Pull request by lpantano**
Update hte environment file to match the last requirements for the next PR I am working on.

### CODE REVIEW

File: environment.yml [hunk 1]
```diff
@@ -6,26 +6,27 @@ channels:
...
-  - conda-forge::r-statmod=1.4.30
-  - conda-forge::r-data.table=1.12.2
-  - conda-forge::r-gplots=3.0.1.1
-  - conda-forge::r-r.methodss3=1.7.1
-  - conda-forge::r-markdown=0.9
-  - conda-forge::matplotlib=3.0.3
+  - seqcluster=1.2.5
   - htseq=0.11.2
   - fastx_toolkit=0.0.14
+  - seqkit
```
Comment (apeltzer) [inline environment.yml]: Please pin this to a specific version ;-) https://bioconda.github.io/recipes/seqkit/README.html

**Comment by lpantano** [comment]
Shoo, I forgot to add that commit, thanks for catching it.


On July 26, 2019 at 6:09:18 AM, Alexander Peltzer (notifications@github.com)
wrote:

*@apeltzer* requested changes on this pull request.
------------------------------

In environment.yml
<https://github.com/nf-core/smrnaseq/pull/20#discussion_r307678352>:

>    - htseq=0.11.2
   - fastx_toolkit=0.0.14
+  - seqkit

Please pin this to a specific version ;-)
https://bioconda.github.io/recipes/seqkit/README.html

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub
<https://github.com/nf-core/smrnaseq/pull/20?email_source=notifications&email_token=AAML6HANFS7SZJP3JGW7VL3QBLEM5A5CNFSM4IG5GCAKYY3PNVWWK3TUL52HS4DFWFIHK3DMKJSXC5LFON2FEZLWNFSXPKTDN5WW2ZLOORPWSZGOB7V223I#pullrequestreview-267103597>,
or mute the thread

**Comment by lpantano** [comment]
@apeltzer, do you think is ok now?

**Comment by drpatelh** [comment]
Yes, its good 👍

**Pull request merged**